datamodel_code_generator/reference.py

```diff
@@ -555,7 +562,12 @@ def join_path(path: Sequence[str]) -> str:
             joined_path += '#'
         return joined_path
 
-    def add_ref(self, ref: str, resolved: bool = False) -> Reference:
+    def add_ref(
+        self,
+        ref: str,
+        resolved: bool = False,
+        http_folder_output: Optional[Path] = None,
```

Comment:
@kevin-lithic 
If `http_folder_output` is always specified, how about adding it to `__init__`?
Of course, if `http_folder_output` is added for the same instance and the case of `None` occurs, you can leave it as is.